src/@utils/subgraph.ts

```diff
@@ -238,6 +250,26 @@ export async function fetchDataForMultipleChains(
   return datas
 }
 
+export async function getOpcFeees(chainId: number) {
```

Comment:
I understand fees are tricky and I also want to scream `feeeees!` but let's use only 2 e's here so it reads `getOpcFees`

Reply:
:)) god one! I'll change it